Python/aot_ceval_jit.c

```diff
@@ -921,20 +1002,19 @@ static void emit_decref(Jit* Dst, int r_idx, int preserve_res) {
     emit_dec_qword_ptr(Dst, &_Py_RefTotal, 1 /* can_use_tmp_reg  */);
 #endif
     | add qword [Rq(r_idx)], -1
-
     // normally we emit the dealloc call into the cold section but if we are already inside it
     // we have to instead emit it inline
     int use_inline_decref = Dst->current_section == SECTION_COLD;
     if (use_inline_decref) {
-        | jnz >8
+        | branch_ne >8 // this is same as 'branch if not zero'
```

Comment:
I always have to check whether ne is nz or z, what do you think about having a `branch_nz` macro as well?